Extension/package.nls.json

```diff
@@ -66,6 +66,12 @@
             "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered."
         ]
     },
+    "c_cpp.configuration.maxSymbolSearchResults.markdownDescription": {
+        "message": "The maximum number of results to show for Go to Symbol in Workspace. The default is `5000`.",
```

Comment:
Can you add '' single quotes around this command? That is the style I've seen most common in the VS Code setting descriptions and we have another example in our package.json.